src/mono/wasm/Wasm.Build.Tests/BrowserRunner.cs

```diff
@@ -119,6 +120,8 @@ public async Task<IBrowser> SpawnBrowserAsync(
         Playwright = await Microsoft.Playwright.Playwright.CreateAsync();
         // codespaces: ignore certificate error -> Microsoft.Playwright.PlaywrightException : net::ERR_CERT_AUTHORITY_INVALID
         string[] chromeArgs = new[] { $"--explicitly-allowed-ports={url.Port}", "--ignore-certificate-errors", $"--lang={locale}" };
+        if (headless)
+            chromeArgs = chromeArgs.Append("--headless=new").ToArray();
```

Comment:
Since Chromium 128 the default mode is headless=new and the "old" mode is headless=old

https://developer.chrome.com/docs/chromium/headless?hl=en
https://issues.chromium.org/issues/346606149

Reply:
We're on `131` https://github.com/dotnet/runtime/blob/92e1cdd4b786ce237f9f378f979e871df3ff076a/eng/testing/BrowserVersions.props

Reply:
Then you should not use --headless=new, but just --headless

Reply:
Just `--headless` results in failures like here
https://github.com/dotnet/runtime/pull/111580#issuecomment-2605693993

Changing to `--headless=new` fixed it.